src/common/config.ts

```diff
@@ -0,0 +1,12 @@
+import { config } from 'dotenv';
+
+config();
```

Comment:
There is an option to use "ConfigService" of @nestjs/config, it provides means to type config values as well.
Under the hood it uses dotenv.

Reply:
Yeap. As I started, first thing about I thought is env variables, even before nestjs doc, at this moment I didn't know there was ConfigService.